src/sagas/index.js

```diff
@@ -27,7 +37,40 @@ function* getTodosSaga() {
   try {
     const todos = yield call(getTodos);
 
-    yield put({type: ADD_TODOS, todos: todos.data});
+    if (todos !== null) {
+      yield put({type: ADD_TODOS, todos: JSON.parse(todos).todos});
```

Comment:
Similarly how we have separated `api` in we might need a wrapper for AsyncStorage. and this `JSON.parse` should go into that.

Reply:
Got it 👍